action/action.go

```diff
@@ -251,7 +251,11 @@ func injectParameters(action string, c *claim.Claim, env, files map[string]strin
 			}
 			continue
 		}
-		value := fmt.Sprintf("%v", rawval)
+		contents, err := json.Marshal(rawval)
+		if err != nil {
+			return err
+		}
+		value := strings.Trim(string(contents), "\"")
```

Comment:
I'm wondering if this `Trim` applied to every contents may alter original content, like a string parameter with double quotes (like `"quoted value"`)?

Reply:
did some quick experiments: 

```
func main() {

	rawval := "[\"first-value\",\"second-value\"]"
	contents, _ := json.Marshal(rawval)
	value := strings.Trim(string(contents), "\"")

	fmt.Printf("Val: %s\n", value)
}
```

That gives `Val: [\"first-value\",\"second-value\"]`

```
func main() {

	rawval := "quoted value"
	contents, _ := json.Marshal(rawval)
	value := strings.Trim(string(contents), "\"")

	fmt.Printf("Val: %s\n", value)
}
```
Gives `Val: quoted value`

```
func main() {

	rawval := "\"escaped quotes\""
	contents, _ := json.Marshal(rawval)
	value := strings.Trim(string(contents), "\"")

	fmt.Printf("Val: %s\n", value)
}
```

Gives: `Val: \"escaped quotes\`

That last one seems like it wouldn't really reflect what the user had passed?

Reply:
Yeah that's not good. I tested this locally as well and rebuilt duffle with and without `strings.Trim`. I came across the same findings as Jeremey where trimming escaped quotes caused unexpected results for the string param. As far as I can tell, removing the `strings.Trim` gives us the desired behavior without causing a problem with escaped quote values. I'm going to update this PR and remove that line.

Reply:
@jeremyrickard @silvin-lubecki it turned out that the simplest thing to do was to keep string values in their raw format. It gives us the desired behavior with strings without having additional logic that tries to figure out when to trim the extra quotes (which felt error prone).